bears/js/PrettierLintBear.py

```diff
@@ -0,0 +1,35 @@
+import re
+from coalib.bearlib.abstractions.Linter import linter
+from dependency_management.requirements.NpmRequirement import NpmRequirement
+
+
+@linter(executable='prettier',
+        use_stdout=True,
+        use_stderr=True)
+class PrettierLintBear:
+    """
+    Formats JavaScript, JSX, Flow, TypeScript, CSS, Less, SCSS, JSON, GraphQL,
+    Markdown files according to opinionated code format
+    using ``prettier``.
+    """
+    LANGUAGES = {'JavaScript'}
```

Comment:
I thought the bear can lint way more languages?

Reply:
It can be used for TypeScript, CSS, JSON, Markdown and many else. I wrote them in the class itself. Do you want me to add them to LANGUAGES object too??

Reply:
yes please :)